### DISCUSSION TITLE
nginx error "host not found" while pushing to other stream services

### DISCUSSION THREAD

**Discussion by bennyevangelista** [tiangolo/nginx-rtmp-docker#80] [Questions]
I just started using nginx-rtmp-docker to test a self-hosted stream server, but when I try to push stream to multiple services (Twitch, facebook, youtube, ..) nginx doesn't start due to error:

`nginx: [emerg] host not found in url "live.twitch.tv/app/......"`

I'm aware this is something related to DNS resolving, and using a real IP the stream is correctly pushed, but I didn't found any solution to this. Is there a workaround?

Here's my nginx.conf file

```
worker_processes auto;
rtmp_auto_push on;
events {}



rtmp {


     server {

        listen 1935;
        listen [::]:1935 ipv6only=on;

        application live{

            live on;
            record off;
            push rtmp://live.twitch.tv/app/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
            
}

```

Am I missing something?

**Comment by nlitz88** [comment]
While I don't receive the "host not found" error, I am also unable to push my incoming streams to other RTMP servers. I am fairly certain it's not due to my nginx configuration. Were you able to get this image working?

**Comment by abhemanyus** [comment]
I'm having the same issue in my docker container cluster
